Homework02/perceptron.py

```diff
@@ -0,0 +1,51 @@
+"""
+perceptron class
+
+Created: 01.11.21, 20:04
+
+Author: LDankert
+"""
+import numpy
+import numpy as np
+
+
+class Perceptron:
+
+    # Constructor
+    def __init__(self, input_units):
+        self.weights = np.random.randn(input_units)
+        self.bias = np.random.randn()
+        self.alpha = 1
+        self.activation = 0
```

Comment:
perhaps, setting activation to zero is not needed - we could remove that line

Reply:
Theoretical true, but instance attributes should always be initialised inside the constructor. Has something to do with readability and memory usage I think.